### PR TITLE
Fix typos in Day0 README.md file

### PR DESCRIPTION
**Pull request by taesup**

### CODE REVIEW

File: Day0/README.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ The following format is meant to be a guideline for effective delivery. Instruct
   1. Don't be "the guy in the room."
```
Comment (mellaniesoriano) [inline Day0/README.md]: The period should come after the quotation mark.

File: Day0/README.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ The following format is meant to be a guideline for effective delivery. Instruct
...
   1. Critique code instead of people
```
Comment (mellaniesoriano) [inline Day0/README.md]: Needs a period at the end of the sentence

File: Day0/README.md [hunk 3]
```diff
@@ -66,15 +66,16 @@ The following format is meant to be a guideline for effective delivery. Instruct
...
 
-1. Set Up Tools (Setup/Live Coding)
+* Set Up Tools (Setup/Live Coding)
   - [Install Fest](https://github.com/devleague/DevLeague-Modules/blob/master/Day0/installFest.md)
   - Chrome
-  - SublimeText, JSHint
+  - Visual Studio Code
     - Discuss the [DevLeague JavaScript Style Guide](https://github.com/devleague/JS-Style-Guide)
   - Terminal, iTerm, Bash, zShell
   - Setup Medium.com Accounts and blogs
```
Comment (mellaniesoriano) [inline Day0/README.md]: Change to "Setup medium.com accounts for blogging"

File: Day0/README.md [hunk 4]
```diff
@@ -66,15 +66,16 @@ The following format is meant to be a guideline for effective delivery. Instruct
...
+  - Setup Free tier AWS account: [here](https://github.com/devleague/DevLeague-Modules/blob/day0-typos/Day0/aws-free-tier-signup.md)
```
Comment (mellaniesoriano) [inline Day0/README.md]: F should be lowercase in Free

File: Day0/README.md (hunk 1 above)
Comment (taesup) [reply Day0/README.md]: I thought periods are before the quotation mark if it ends the sentence?

**Comment by mellaniesoriano** [comment]
LGTM ✅

**Pull request merged**